apps/jobs/views.py

```diff
@@ -901,36 +911,22 @@ def update_submission(request, challenge_pk):
                 leaderboard_metrics = challenge_phase_split.leaderboard.schema.get(
                     "labels"
                 )
-                missing_metrics = []
-                malformed_metrics = []
-                for metric, value in accuracies.items():
-                    if metric not in leaderboard_metrics:
-                        missing_metrics.append(metric)
-
-                    if not (
-                        isinstance(value, float) or isinstance(value, int)
-                    ):
-                        malformed_metrics.append((metric, type(value)))
-
-                if len(missing_metrics):
-                    response_data = {
-                        "error": "Following metrics are missing in the"
-                        "leaderboard data: {}".format(missing_metrics)
-                    }
-                    return Response(
-                        response_data, status=status.HTTP_400_BAD_REQUEST
-                    )
 
-                if len(malformed_metrics):
-                    response_data = {
-                        "error": "Values for following metrics are not of"
-                        "float/int: {}".format(malformed_metrics)
-                    }
-                    return Response(
-                        response_data, status=status.HTTP_400_BAD_REQUEST
-                    )
-
-                data = {"result": accuracies}
+                for field in [
```

Comment:
can we use `zip` method here?

Reply:
If you are saying as using
```py
zip((leaderboard_metrics, accuracies), (["error_{0}".format(i) for i in leaderboard_metrics], errors))
```
instead of 
```py
[(leaderboard_metrics, accuracies), (["error_{0}".format(i) for i in leaderboard_metrics], errors)]
```
I think we cannot, as zip doesnt help here.
for example, this is desired for `validate_numerical_values_in_result`:
```
(['metric1', 'metric2'], {'metric1': 60, 'metric2': 30})
(['error_metric1', 'error_metric2'], {'error_metric1': 3, 'error_metric2': '8'})
```

but using zip, we get this:
```
(['metric1', 'metric2'], ['error_metric1', 'error_metric2'])
({'metric1': 60, 'metric2': 30}, {'error_metric1': 3, 'error_metric2': '8'})
```